CDP4Common/Poco/NestedElement.cs

```diff
@@ -74,5 +74,82 @@ protected DomainOfExpertise GetDerivedOwner()
         /// is the root of the Nested tree.
         /// </summary>
         public bool IsRootElement { get; internal set; }
+
+        /// <summary>
+        /// The <see cref="ElementBase"/> representing a <see cref="NestedElement"/>.
+        /// </summary>
+        /// <returns>
+        /// The <see cref="ElementBase"/>.
+        /// </returns>
+        public ElementBase GetElementBase()
+        {
+            return this.IsRootElement
+                ? (ElementBase)this.RootElement
+                : this.ElementUsage.Last();
+        }
+
+        /// <summary>
+        /// The <see cref="ElementDefinition"/> representing a <see cref="NestedElement"/>, if it exists.
+        /// </summary>
+        /// <returns>
+        /// The <see cref="ElementDefinition"/>.
+        /// </returns>
+        public ElementDefinition GetElementDefinition()
+        {
+            var elementBase = this.GetElementBase();
+
+            return elementBase as ElementDefinition ?? (elementBase as ElementUsage)?.ElementDefinition;
+        }
+
+        /// <summary>
+        /// The <see cref="ElementUsage"/> representing a <see cref="NestedElement"/>, if it exists.
+        /// </summary>
+        /// <returns>
+        /// The <see cref="ElementUsage"/>.
+        /// </returns>
+        public ElementUsage GetElementUsage()
+        {
+            return this.GetElementBase() as ElementUsage;
+        }
+
+        /// <summary>
+        /// Checks if a <see cref="NestedElement"/> contains a specific <see cref="Category"/>.
+        /// </summary>
+        /// <param name="category">
+        /// The <see cref="Category"/>.
+        /// </param>
+        /// <returns>
+        /// True if the <paramref name="category"/> is found, otherwise false.
+        /// </returns>
+        public bool HasCategory(Category category)
```

Comment:
The name of the method should also be changed. IsMemberOf... then it is semantically right